app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -333,6 +351,13 @@ function YourTurnContent(props: {
             </Grid>
           )}
         </Grid>
+
+        {!isMobile &&
+          props.activeTurnPlayState === ActiveTurnPlayState.Playing && (
+            <div style={{ textAlign: "center", color: grey[500] }}>
+              [ Hint: Press the spacebar for "Correct", and S for "Skip" ]
+            </div>
```

Comment:
```suggestion
            <Grid item style={{ color: grey[500] }}>
              Hint: Press the <span className="code">spacebar</span> for
              "Correct", and S for "Skip"
            </Grid>
```

Reply:
removed the [ ] just to make it display nicer on Safari 🤦  hope you don't mind

Reply:
and removed text align for stylistic similarity, switch to grid item for the layout defaults